config/overlays/test/configmap/inferenceservice.yaml

```diff
@@ -36,7 +36,7 @@ data:
           }
         },
         "pytorch": {
-            "image": "527798164940.dkr.ecr.us-west-2.amazonaws.com/kfserving/pytorchserver",
+            "image": "pytorch/torchserve",
```

Comment:
the test config needs to change to follow the same v1/v2 ProtocolConfig as above.